frontend/src/views/GShootList.vue

```diff
@@ -704,6 +705,9 @@ export default {
       const items = this.sortItems(this.items, this.sortByInternal)
       return filter(items, item => this.searchItems(this.debouncedShootSearch, toRaw(item)))
     },
+    showOperatorFeatures () {
```

Comment:
rename to `issueSinceColumnVisible` / ``issueSinceColumnHidden`?

Reply:
renamed to `issueSinceColumnVisible`